REVIEWING.md

```diff
@@ -0,0 +1,84 @@
+# Reviewing pull requests
+
+This document provides a high-level and **incomplete** checklist of things that pull request reviewers should be aware of when performing code review. The checklist mainly consists of things that are specific to the scikit-bio project and that generally apply to incoming pull requests. The checklist is incomplete because it is not possible to describe all things to verify during code review (that depends on what is being reviewed). This document also doesn't attempt to describe *how* to perform code review (there are many online resources for that).
+
+Reviewers are encouraged to keep this document up-to-date as the project evolves and to add anything that's missing.
+
+The checklist is not in any particular order.
+
+## Changelog
+
+This is one of the easiest parts to forget in a pull request.
+
+- Note all public (i.e. user-facing) changes in `CHANGELOG.md` under the latest development version section at the top of the file. This includes things like bug fixes, API additions/changes/removal, performance enhancements, etc. The changelog has several subsections for organizing these changes.
```

Comment:
There isn't a way to link to bullet points within the rendered markdown. I think the changelog as a whole serves as an example.